maintainers/maintainer-list.nix

```diff
@@ -8039,6 +8039,12 @@
     githubId = 3708689;
     name = "Roberto Di Remigio";
   };
+  robertoszek = {
+    email = "robertoszek@robertoszek.xyz";
+    github = "robertoszek";
+    githubId = 1080963;
+    name = "Roberto";
+  };
```

Comment:
Can you split this in a different commit with the message `maintainers: add robertoszek`?

Reply:
Done. Apologies for bundling it into the same commit.